### ISSUE TITLE
Not working in mobile browsers?

### ISSUE THREAD

**Issue by BlackSnail** [jlmakes/scrollreveal.js#208]
i want have some animate in mobile browser when some elements show. The scrollreveal althought 
very well in computer browser, but it no animate in mobile browser.
could it be support animate in mobile browser ?

**Comment by mikerockett** [comment]
There are many mobile browsers. Which one are you referring to?

**Comment by jlmakes** [comment]
@BlackSnail Yes, please provide more details… the device, operating system and browser. :bee:

**Comment by AyanGhatak** [comment]
Hey @jlmakes I think this is a valid issue. I have been trying on IOS Safari 8 and also in Chrome version in ipad. Things dont work even in (samsung galaxy tab/android/firefox, etc.) There is a easing effect but not the rotate,delay and other effects. Same things work smooth on my desktop browser.This is the configurations I used:

window.sr = ScrollReveal().reveal('.column', {
		duration: 2000,
		delay: 10,
		origin: 'left',
		rotate: { x: 30, y: 70, z: 60 },
		scale: 0.5,
		mobile: true
	});

I tried debugging in safari and it seems there is some issue with webkit-transform on ios. Give me a couple of days.Hope to send you a PR once done. But this is @Confirmed issue.

**Comment by jlmakes** [comment]
Thank you @AyanGhatak!

I noticed a problem thanks to your feedback; not too long ago I noticed I had my `-webkit-` prefixed styles before the un-prefixed styles… and so I flipped the order in which they are generated ([this commit](eb3295e)), but it seems I forgot to update the operators!

_See my fix below:_
```diff
- elem.styles.transform.initial += ' -webkit-transform:';
- elem.styles.transform.target  += ' -webkit-transform:';
+ elem.styles.transform.initial = ' -webkit-transform:';
+ elem.styles.transform.target  = ' -webkit-transform:';
generateTransform( elem.styles.transform );

- elem.styles.transform.initial = 'transform:';
- elem.styles.transform.target  = 'transform:';
+ elem.styles.transform.initial += 'transform:';
+ elem.styles.transform.target  += 'transform:';
generateTransform( elem.styles.transform );
```

**Comment by AyanGhatak** [comment]
Hey @jlmakes , I think the order of concatenation of the prefixes was not proper. Please review the [this commit](https://github.com/AyanGhatak/scrollreveal.js/commit/c7b29faf83236eeff2210250ea1bce3a51dda576). I have tested with leading browsers in android and ios platform and it works fine. Please validate.

**Comment by jlmakes** [comment]
@AyanGhatak The vendor prefixes _should_ come first, but you’re referencing the same bug I fixed in c01def2 (the previous comment)

The difference is that I didn’t rebuild the `/dist` folder. If you are to run `gulp build` on the latest master, you should find the same results as your changes—but with vendor prefixes correctly before the un-prefixed styles.

**Comment by jlmakes** [comment]
@AyanGhatak You’ve changed the `/dist/scrollreveal.js` manually, but this was already fixed in the source file in a recent commit (c01def2). I left a more detailed comment on the PR.

I’ve released `v3.0.4`, please confirm it is working as expected—particularly in the browsers you noticed with broken transforms:
>I have been trying on IOS Safari 8 and also in Chrome version in iPad. Things don’t work even in (samsung galaxy tab/android/firefox, etc.)

I presume this is only part of the fix, as iOS Chrome animations work—but the library still fails to animate reveal properly on scroll. I could use your help troubleshooting #196. :bow:
